### PR TITLE
Create Docker Compose Controller Artifact

### PR DESCRIPTION
**Pull request by adamkroonbcgov**
## 🎯 Summary

[DEVOP-235](https://citz-imb.atlassian.net/jira/software/c/projects/DEVOP/boards/18?selectedIssue=DEVOP-235)

## 🔰 Checklist

- [x] I have read and agree with the following checklist

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.

[DEVOP-235]: https://citz-imb.atlassian.net/browse/DEVOP-235?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .docker/docker-compose.controller.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      context: database/mssql
+    env_file:
+      - database/mssql/.env
+    ports:
+      - ${DATABASE_PORT:-5433}:1433
+    volumes:
+      - database-data:/var/opt/mssql/data
+    networks:
+      - ${NAME_OF_PRODUCT}-network
+  ########################### End MYSQL Database ##############################
```
Comment (adamkroonbcgov) [inline .docker/docker-compose.controller.yaml]:
```suggestion
```

File: .docker/docker-compose.controller.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    build:
+      context: react-app
+      target: prod
+    ports:
+      - ${APP_HTTP_PORT:-3000}:3000
+    depends_on:
+      - express-api
+    env_file: .env
+    networks:
+      - z${NAME_OF_PRODUCT}-network
```
Comment (adamkroonbcgov) [inline .docker/docker-compose.controller.yaml]:
```suggestion
      - ${NAME_OF_PRODUCT}-network
```

**Review by Bottle7 — DISMISSED** [review]

File: .docker/docker-compose.controller.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    healthcheck:
+      test: ['CMD-SHELL', 'pg_isready -d $${POSTGRES_DB} -U $${POSTGRES_USER} || exit 1']
+      interval: 5s
+      timeout: 3s
+    networks:
+      - ${NAME_OF_PRODUCT}-network
+  ########################### End Postgres ####################################
+
+  ############################# Express API ###################################
+  ${NAME_OF_PRODUCT}-api:
```
Comment (adamkroonbcgov) [inline .docker/docker-compose.controller.yaml]:
```suggestion
  api:
```

File: .docker/docker-compose.controller.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    restart: 'no'
+    container_name: ${NAME_OF_PRODUCT}-api
```
Comment (adamkroonbcgov) [inline .docker/docker-compose.controller.yaml]:
```suggestion
    container_name: api
```

**Comment by Bottle7** [comment]
Type for the network on the react-app network spec

**Review by Bottle7 — APPROVED** [review]

**Pull request merged**